articles/azure-functions/functions-scenario-database-table-cleanup.md

```diff
@@ -81,7 +83,7 @@ Now, you can add the C# function code that connects to your SQL Database.
     using System.Threading.Tasks;
 	```
 
-4. Replace the existing **Run** function with the following code:
+4. Replace the existing `Run` function with the following code:
```

Comment:
`Run` is the name of the method, so it should be formatted as code

Reply:
Sadly, we don't actually have good guidance for docs.microsoft.com on how styles apply to various types of APIs. I prefer **bold** for objects/methods, but will accept the change to `code` style.